dashboard/15-final/app/dashboard/invoices/page.tsx

```diff
@@ -1,9 +1,16 @@
 import InvoicesTable from '@/app/ui/invoices/table';
 
-export default function Page() {
+export default function Page({
+  searchParams,
+}: {
+  searchParams: {
+    q: string;
```

Comment:
Small nit, I know q is quite common, but for beginners, I think `query` would be better so it's self-explanatory.